File: research/stellar/examples/javascript/account/fund-account.js
const StellarSdk = require('stellar-sdk')

var argv = require('yargs')
    .usage('Usage: $0 --address [string]')
    .demandOption(['address'])
    .argv;


function fundThroughFriendbot (address) {
    return StellarSdk.HorizonAxiosClient.get("https://friendbot.stellar.org/?addr=" + address)
        .then(res => {
        if (res.status === 200) {
            console.log(`account with address: ${address} funded through friendbot!`)
        }
        })
        .catch(err => {
        console.log(err)
        return err
        })
}


fundThroughFriendbot(argv.address)

File: research/stellar/examples/python/account/fund-account.py
# pylint: disable=no-value-for-parameter
import requests
import sys, getopt
import click

@click.command()
@click.option('--address', help='Address to fund', required=True)

def fundThroughFriendbot(address):
  try:
    res = requests.get("https://friendbot.stellar.org/?addr=" + address)
    res.raise_for_status()
    print("account with address: {} funded through friendbot".format(address))
  except requests.exceptions.HTTPError:
    print(res.json())
    sys.exit(1)

if __name__ == '__main__':
  fundThroughFriendbot()